app/src/com/android/rkpdapp/provisioner/WidevineProvisioner.java

```diff
@@ -161,9 +162,14 @@ public Result provisionWidevine() {
 
     private byte[] fetchWidevineCertificate(MediaDrm.ProvisionRequest req) throws IOException {
         final byte[] data = req.getData();
+        String url = req.getDefaultUrl();
+        String override = WidevineProvisioningSettings.getServerUrlOverride(getApplicationContext());
+        if (override != null) {
+            url = url.replaceAll("www.googleapis.com", override);
```

Comment:
This line will be a no-op if URL doesn't contain `www.googleapis.com`.
Need to replace specifically hostname from `req.getDefaultUrl()`.